sass/components/_timeperiod.scss

```diff
@@ -195,7 +195,7 @@
         top: 0px;
 
         > .cq-menu-btn {
-            padding: 16px 5px !important;
+            // padding: 16px 5px !important;
```

Comment:
@ashkanx if the padding is not needed we can just remove it  and please remove it fot ` .smartcharts-480 ` as well. its in line 190